perf/index.js

```diff
@@ -0,0 +1,47 @@
+const Benchmark = require('benchmark');
+const fc = require('../lib/fast-check');
+
+const benchSettings = function(label, arb) {
+    (new Benchmark.Suite())
+        .add(`[default] ${label}`, () => fc.sample(arb))
+        .add(`[unbiased] ${label}`, () => fc.sample(arb, { unbiased: true }))
+        .add(`[random:congruential] ${label}`, () => fc.sample(arb, { randomType: 'congruential' }))
+        .add(`[random:congruential32] ${label}`, () => fc.sample(arb, { randomType: 'congruential32' }))
+        .on('cycle', (event) => console.log(String(event.target)))
+        .on('complete', function() { console.log(`Fastest is ${this.filter('fastest').map('name')}\n`); })
+        .run({ async: false });
+};
+
+benchSettings('boolean', fc.boolean());
+benchSettings('integer', fc.integer());
+benchSettings('integer (min, max)', fc.integer(Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER));
+benchSettings('float', fc.float());
+benchSettings('double', fc.double());
+benchSettings('array', fc.array(fc.integer()));
+benchSettings('array (min size)', fc.array(fc.integer(), 5, 10));

```

Comment:
5 and 10 look like magic number here

Reply:
This code in neither a production code nor a test code, it is used to run performance measurements on the library. Those 'hardcoded' values have been used to take into account cases with an array having a minimal size.

For the moment, specifying a minimal size forces the user to specify a maximal one. This problem should be fixed in a later iteration thanks to #89.